nri-redis/src/Redis/Settings.hs

```diff
@@ -55,13 +56,24 @@ decoder :: Environment.Decoder Settings
 decoder =
   decoderWithEnvVarPrefix ""
 
+-- | decodes Settings from environmental variables with custom connection string
```

Comment:
Would this be better @jali-clarke ?

```suggestion
-- | decodes Settings from environmental variables with specific name for the connection string variable
```

Other than that I find it hard to include something like, because we setup rails using an env var that does not follow our haskell services conventions and it's either this or adding a duplicated configuration. Which does not helps for public documentation 🙈

Reply:
yup all good - i'm all for adding this configurable, just wanted it to be clear as to why it exists in the first place